src/fontstash.h

```diff
@@ -781,6 +822,12 @@ void fonsSetFont(FONScontext* stash, int font)
 	fons__getState(stash)->font = font;
 }
 
+void fonsSetOversample(FONScontext* stash, int oversampleH, int oversampleV)
+{
+	fons__getState(stash)->oversampleH = oversampleH;
```

Comment:
We probably should validate oversampling here. I.e. make sure it is between 1 and 16 or something like that.